src/Test/Unit/Config/Validator/Deploy/DatabaseSplitConnectionTest.php

```diff
@@ -78,14 +79,19 @@ public function testMessageValidate()
             ->willReturn($dbConfiguration);
         $this->resultFactoryMock->expects($this->once())
             ->method('error')
-            ->with('Split database configuration was detected in the property DATABASE_CONFIGURATION of the'
-                . ' file .magento.env.yaml:' . PHP_EOL
-                . '- connection: checkout' . PHP_EOL
-                . '- connection: sales' . PHP_EOL
-                . '- slave_connection: checkout' . PHP_EOL
-                . '- slave_connection: sales' . PHP_EOL
-                . 'Magento Cloud does not support a custom split database configuration,'
-                . ' such configurations will be ignored');
+            ->with(
+                'Split database configuration was detected in the property DATABASE_CONFIGURATION of the'
+                    . ' file .magento.env.yaml:' . PHP_EOL
+                    . '- connection: checkout' . PHP_EOL
+                    . '- connection: sales' . PHP_EOL
+                    . '- slave_connection: checkout' . PHP_EOL
+                    . '- slave_connection: sales' . PHP_EOL
+                    . 'Magento Cloud does not support a custom split database configuration,'
```

Comment:
Not totally clear on this error message-- Is it:  Magento Cloud does not support custom connections in the split database configuration?

Or you can't include split db configuration in the DATABASE_CONFIGURATION?

Reply:
Customer cannot set custom split connections

Reply:
This means that we do not support split database settings with third-party database servers